src/depth-to-rgb-calibration.cpp

```diff
@@ -159,8 +168,12 @@ rs2_calibration_status depth_to_rgb_calibration::optimize(
         // cost= " << params_curr.cost );
 
         AC_LOG( DEBUG, "Optimization successful!" );
-        _intr = _algo.get_calibration().get_intrinsics();
-        _intr.model = RS2_DISTORTION_INVERSE_BROWN_CONRADY; //restore LRS model 
+        _intr_with_k_thermal = _algo.get_calibration().get_intrinsics();
+
+        // override only the ppx and ppy the fx and fy are the original
```

Comment:
Why not copy `K_th` and restore `fx` anf `fy`, instead? That way we do not assume we know what changed or didn't change...

Reply:
_intr saves the original fx and fy

Reply:
The difference is:
```
_raw.ppx = _thermal.ppx;
// ... and repeat for each and every member that needs copying
// Assumes we know what changed in the intrinsics -- for example, that `model` and `coeffs` can't change
```
vs:
```
double original_fx = _raw.fx, original_dy = _raw.fy;
_raw = _thermal;
_raw.fx = original_fx;
_raw.fy = original_fy;
// Everything else changes
```

Reply:
done